R/misc.R

```diff
@@ -266,3 +266,20 @@ has.function <- function (query, funcs) {
 
     return(FALSE)
 }
+
+escapeQuotes <- function(str) {
```

Comment:
Why does this escape both single and double quotes? Seems like you only need to escape whatever you're using as the string quote in `buildLoadDatasetCall` (which appears to be `'`.